packages/account-v2/src/modules/TradingAssessmentForm/__test__/RiskToleranceWarningModal.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Modal } from '@deriv-com/ui';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { ACCOUNT_MODAL_REF } from '../../../constants';
+import { RiskToleranceWarningModal } from '../RiskToleranceWarningModal';
+
+describe('RiskToleranceWarningModal', () => {
+    let elModalRoot: HTMLElement;
+    beforeAll(() => {
+        elModalRoot = document.createElement('div');
+        elModalRoot.setAttribute('id', ACCOUNT_MODAL_REF.replace('#', ''));
+        document.body.appendChild(elModalRoot);
+        Modal.setAppElement(ACCOUNT_MODAL_REF);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(elModalRoot);
+    });
+    it('should renders correctly with the modal open', () => {
+        const handleSubmit = jest.fn();
+        const isModalOpen = true;
+
+        render(<RiskToleranceWarningModal handleSubmit={handleSubmit} isModalOpen={isModalOpen} />);
+
+        expect(screen.getByText('Risk Tolerance Warning')).toBeInTheDocument();
+
+        expect(
+            screen.getByText(
+                /CFDs and other financial instruments come with a high risk of losing money rapidly due to leverage/
+            )
+        ).toBeInTheDocument();
+
+        expect(
+            screen.getByText(/To continue, you must confirm that you understand your capital is at risk/)
```

Comment:
I see the text is appearing in more than 1 place :)
let's. move it to separate variable :)